odoo/tools/cache.py

```diff
@@ -216,26 +143,30 @@ def log_ormcache_stats(sig=None, frame=None):
     me = threading.current_thread()
     me_dbname = getattr(me, 'dbname', 'n/a')
 
-    for dbname, reg in sorted(Registry.registries.d.items()):
-        # set logger prefix to dbname
-        me.dbname = dbname
-        entries = Counter(k[:2] for k in reg._Registry__cache.d)
+    def _log_ormcache_stats(cache_name, cache):
+        entries = Counter(k[:2] for k in cache.d)
         # show entries sorted by model name, method name
         for key in sorted(entries, key=lambda key: (key[0], key[1].__name__)):
             model, method = key
             stat = STAT[(dbname, model, method)]
             _logger.info(
-                "%6d entries, %6d hit, %6d miss, %6d err, %4.1f%% ratio, for %s.%s",
-                entries[key], stat.hit, stat.miss, stat.err, stat.ratio, model, method.__name__,
+                "%s, %6d entries, %6d hit, %6d miss, %6d err, %4.1f%% ratio, for %s.%s",
```

Comment:
keep fixed size to have readable log

Reply:
I added a rjust(25) on cache_name if it is ok for you.